2-core-rules.md

```diff
@@ -490,43 +474,43 @@ The higher a weapon's Strength, the more easily it can pierce armor. The chart b
 | 8 | - |
 | 9+ | - |
 
-Some weapons are better at penetrating armor than their Strength value suggests (Elven bows, for example). This is covered in the entry for each particular weapon (see the Weapons & Armor section). 
+Some weapons are better at penetrating armour than their Strength value suggests (Elven bows, for example). This is covered in the entry for each particular weapon (see the Weapons & Armour section). 
 
-_Example: Dieter wears heavy armor and carries a shield. His armor save is 4+. He is hit by a crossbow (Strength 4) and therefore he will save on a D6 roll of 5+ (i.e., 4+ 1 = 5+)._
+_Example: Jeremy wears heavy armour and carries a shield. His armour save is 4+. He is hit by a crossbow (Strength 4) and therefore he will save on a D6 roll of 5+ (i.e., 4+ 1 = 5+)._
```

Comment:
```suggestion
_Example: Useless Jeremy wears heavy armour and carries a shield. His armour save is 4+. He is hit by a crossbow (Strength 4) and therefore he will save on a D6 roll of 5+ (i.e., 4+ 1 = 5+)._
```

Reply:
Dammit I just wanted to be nice to Jeremy for once.